client/my-sites/sidebar-unified/item.jsx

```diff
@@ -36,13 +39,24 @@ export const MySitesSidebarUnifiedItem = ( {
 	continueInCalypso,
 } ) => {
 	const reduxDispatch = useDispatch();
-	const isHappychatSessionActive = useSelector( ( state ) => hasActiveHappychatSession( state ) );
+	const isHappychatSessionActive = useSelector( hasActiveHappychatSession );
+	const siteId = useSelector( getSelectedSiteId );
+	const isJetpack = useSelector( ( state ) => isJetpackSite( state, siteId ) );
+	const isSiteAtomic = useSelector( ( state ) => isSiteWpcomAtomic( state, siteId ) );
+	const isJetpackNonAtomicSite = isJetpack && ! isSiteAtomic;
 
 	const onNavigate = () => {
 		reduxDispatch( collapseAllMySitesSidebarSections() );
 		reduxDispatch( expandMySitesSidebarSection( sectionId ) );
 	};
 
+	let forceInternalLink = true;
+	if ( isHappychatSessionActive && ! isJetpackNonAtomicSite ) {
```

Comment:
On a similar note to https://github.com/Automattic/wp-calypso/pull/50444/files#r582802212

I would remove `&& ! isJetpackNonAtomicSite` from here.

Reply:
Yup, good catch. Done in https://github.com/Automattic/wp-calypso/pull/50444/commits/3c8c6d27ef3957a03beea37633a33e0fbeac7c09.